### PR TITLE
Max record bug fix

### PR DESCRIPTION
**Pull request by juggernautsei**
As requested

### CODE REVIEW

File: interface/new/new_comprehensive_save.php [hunk 1]
```diff
@@ -28,11 +28,9 @@
 // other interfaces can still read the data during this lock, however
 // sqlStatement("lock tables patient_data read");
 
-$result = sqlQuery("SELECT MAX(pid)+1 AS pid FROM patient_data");
-
+$result = sqlQuery("SELECT id AS pid FROM patient_data ORDER BY id DESC LIMIT 1");
```
Comment (bradymiller) [inline interface/new/new_comprehensive_save.php]: I don't think id should be used here. Shouldn't this instead be:
``````
$result = sqlQuery("SELECT pid FROM patient_data ORDER BY pid DESC LIMIT 1");
``````

Comment (bradymiller) [inline interface/new/new_comprehensive_save.php]: Also have changes in 2 unrelated files here:
interface/documents/upload.php
templates/documents/general_upload.html

Comment (juggernautsei) [inline interface/new/new_comprehensive_save.php]: I reverted those files, didn't I?
I'll change it to pid.

Comment (bradymiller) [inline interface/new/new_comprehensive_save.php]: @juggernautsei , the only script that should be touched in this PR is interface/new/new_comprehensive_save.php

Comment (juggernautsei) [inline interface/new/new_comprehensive_save.php]: I will build a new PR.
